lib/plistLib/main.js

```diff
@@ -127,7 +127,8 @@ define([
             return doc;
         },
         getType: function(value) {
-            if(value instanceof IntObject)
+            if(value instanceof IntObject
+                    || typeof value === 'number' && value === (value|0))
```

Comment:
this looks like it's missing parentheses, unless you intended both `typeof value === 'number' && value === (value|0)` and `value instanceof IntObject && value === (value|0)` to be necessary. Otherwise this probably wants to be
```
if(value instanceof IntObject
    || (typeof value === 'number' && value === (value|0)))
```